README.rst

```diff
@@ -503,8 +503,83 @@ Models
+
+               -  ``Yes``
+
+                  #. ``Specify the maximum allowed mean framewise displacement in mm`` By selecting 0.5, this will exclude all subjects (or runs/sessions for a subject if ‘aggregation of scan-level statistics before analysis’ is not selected above) with a mean framewise displacement (root-mean-square) >0.5 mm. If aggregation of scan-level statistics before analysis is selected, the aggregate image will be used in the group-level analyses, therefore the average framewise displacement of all images used for the aggregate image will be compared to the value set here – if it is exceeded, the subject will be excluded from analysis.
+                  #. ``Specify the maximum allowed percentage of frames above the framewise
+                     displacement threshold of (FD max from above)``
+
+               -  ``No`` All subjects are retained for analyses regardless of movement.
+
+         -  ``Linear model`` This will apply a linear model that captures main and/or interaction effects of one or more variables that must be present in a covariate/group data spreadsheet file. This file must contain a column with subject ID names identical to those used for the functional data.
+
+            #. ``Specify model name``
+            #. ``Aggregate scan-level statistics before analysis?`` See above
+            #. ``Specify the path of the covariates/group data spreadsheet file`` Can be a .csv, .txt, .tsv, or .xlsx file
+
+               #. ``Specify the column containing subject names``
+               #. ``Specify the column data types`` Note that the black option is selected, the grey option is not. Toggle left/right keys to select or deselect options and up/down keys to move through the list of variables in your spreadsheet file.
+               #. ``Specify the subjects to use`` Select the subjects to include in this analysis by their categorical variables. For multiple categorical variables, the intersection of the groups will be used.
+
+            #. ``Exclude subjects based on movement?`` See above
+
+               -  ``Yes``
+
+                  #. ``Specify the maximum allowed mean framewise displacement in mm``
+                  #. ``Specify the maximum allowed percentage of frames above the framewise
+                     displacement threshold of (FD max from above)``
+
+               -  ``No`` Continue
+
+            #. ``Specify the variables to add to the model`` For all variables that are selected here, the model will output a main effect, controlling for all other variables selected by treating them as covariates of no interest. Anything not selected at this stage will be left entirely out of the model.
+
+               -  ``Listwise deletion`` Will remove subjects with a missing value from analysis entirely.
+               -  ``Mean substitution`` For continuous variables, will impute the average of all subjects with a valid observation for this variable.
```

Comment:
I think we also do mean substitution for categorical variables (after dummy coding). Maybe we should switch that to mode imputation. What do you think? Also @ilyaveer

Reply:
Resolved